libs/widget-lib/src/types.ts

```diff
@@ -150,6 +151,58 @@ interface CowSwapWidgetConfig {
    * Please contact https://cowprotocol.typeform.com/to/rONXaxHV
    */
   interfaceFeeBips: string
+
+  /**
+   * Sounds configuration for the app.
+   */
+  sounds?: {
+    /**
+     * The sound to play when the order is executed. Defaults to world wide famous CoW Swap moooooooooo!
+     * Alternatively, you can use a URL to a custom sound file, or set to null to disable the sound.
+     */
+    postOrder?: string | null
+
+    /**
+     * The sound to play when the order is executed. Defaults to world wide famous CoW Swap happy moooooooooo!
+     * Alternatively, you can use a URL to a custom sound file, or set to null to disable the sound.
+     */
+    orderExecuted?: string | null
+
+    /**
+     * The sound to play when the order is executed. Defaults to world wide famous CoW Swap unhappy moooooooooo!
+     * Alternatively, you can use a URL to a custom sound file, or set to null to disable the sound.
+     */
+    orderError?: string | null
+  }
+
+  /**
+   * Customizable images for the widget.
+   */
+  images: {
+    /**
+     * The image to display when the orders table is empty (no orders yet). It defaults to "Yoga CoW" image.
+     * Alternatively, you can use a URL to a custom image file, or set to null to disable the image.
+     */
+    emptyOrders?: string | null
+  }
+
+  /**
+   * Disables showing the confirmation modal you get after posting an order.
+   * Defaults to false.
+   */
+  disablePostedOrderConfirmationModal?: boolean
+
+  /**
+   * Disables showing the toast messages.
+   * Some UI might want to disable it and subscribe to 'onToastMessage' event to handle the toast messages itself.
+   * Defaults to false.
+   */
+  disableToastMessages?: boolean
+
+  /**
+   * Disables showing the recent history in the widget. Defaults to false.
+   */
+  hideAccountButton?: boolean
```

Comment:
The comment to me indicates only the recent history is shown, but from the property name I take the whole account button will be removed.

Reply:
not sure how to call it :) 

<img width="642" alt="image" src="https://github.com/cowprotocol/cowswap/assets/2352112/fc362848-b288-4d19-9512-5e1dcadeb171">

<img width="1317" alt="image" src="https://github.com/cowprotocol/cowswap/assets/2352112/a89a5c55-87e3-4e4d-b117-594255848d9e">


I will go for `hiddeConnectButton`

Reply:
52b007c7327172f71d67b38953f1b55d6e68dc81

Reply:
The property name was fine IMO, the comment was more about the description not matching it.

Reply:
I think the new name is more general. I hope the description matches too, let me know if its all good